code/api/src/main/scala/com/avast/datadog4s/api/metric/Eventer.scala

```diff
@@ -0,0 +1,37 @@
+package com.avast.datadog4s.api.metric
+
+import com.avast.datadog4s.api.Tag
+import com.avast.datadog4s.api.event.{ AlertType, Event }
+
+trait Eventer[F[_]] {
+
+  def send(
+    event: Event,
+    tags: Tag*
+  ): F[Unit]
+
+  def error(
+    event: Event,
+    tags: Tag*
+  ): F[Unit] =
+    send(event.copy(alertType = AlertType.Error), tags: _*)
```

Comment:
I don't like the pattern of overriding something that user can set - i think it introduces some complexity into the expected behaviour.

Reply:
I think with the proposed event encoding above, we can get away with just one method `send` and it makes the code simpler to use

Reply:
sounds better 😄